playbooks/templates/webhooktranslator_service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Webhook translator service (MaaS to Jira)
+After=network.target
+
+[Service]
+Environment="RACKSPACE_WEBHOOK_TOKEN={{RACKSPACE_WEBHOOK_TOKEN}}" "JIRA_USER={{JIRA_USER}}" "JIRA_PASSWORD={{JIRA_PASSWORD}}" "JIRA_PROJECT={{JIRA_PROJECT}}"
```

Comment:
Decided not to do this as some of the variables come from environment variables so make sense to be capitalised, and the others are capitalised for consistency.